corehq/apps/geospatial/reports.py

```diff
@@ -53,6 +40,24 @@ def template_context(self):
     def report_context(self):
         cases = []
         invalid_geo_cases_count = 0
+
+        def _get_geo_location(case):
```

Comment:
you could use the [`GeoPoint`](https://github.com/dimagi/commcare-hq/blob/663ba850feae022a552202e814a579782358fe04/corehq/ex-submodules/couchforms/geopoint.py#L8) class to parse the property value